schemas/recipeNew.json

```diff
@@ -0,0 +1,226 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "http://example.com/example.json",
+    "type": "object",
+    "title": "The root schema",
+    "description": "The root schema comprises the entire JSON document.",
+    "default": {},
+    "examples": [{
+        "Name": "Test Recipe",
+        "ABV": 5.6,
+        "OG": 1.062,
+        "FG": 1.01,
+        "IBU": 77,
+        "public": false,
+        "shareable": false,
+        "style": "IPA",
+        "active": true,
+        "instructions": "Blank list of instructions",
+        "Ingredients": [{
+                "key": 1,
+                "Name": "Munich Malt",
+                "quantity": 4
+            },
+            {
+                "key": 2,
+                "Name": "Citra",
+                "quantity": 2
+            },
+            {
+                "key": 3,
+                "Name": "Water",
+                "quantity": 7
+            }
+        ]
+    }],
+    "required": [
+        "Name",
+        "ABV",
+        "OG",
+        "FG",
+        "IBU",
+        "public",
+        "shareable",
+        "style",
+        "active",
+        "instructions",
+        "Ingredients"
+    ],
```

Comment:
Looks like we need to update to styleId and userid. The updated definitions should be in main

Reply:
Changes have been made to incorporate styleId and userId into the create recipe logic.